### PR TITLE
Create 190818-qingjiao

### PR DESCRIPTION
**Pull request by GUThe**
#63

### CODE REVIEW

**Comment by GUThe** [comment]
不知道为什么写辣么久。。。
是我太菜了吗？

File: subtitles/wang-gang/190818-qingjiao [hunk 1]
```diff
@@ -0,0 +1,196 @@
+虎皮青椒 - https://youtu.be/dvl2yq25nlU
+这是刚哥最新的视频私享链接，方便大家提前添加字幕，做菜过程文字稿在下面
+文字稿可能会和视频有微小出入，请大家以视频为主。然后就是由于刚哥的合同问题，请大家务必不要泄露或分享刚哥未在YouTube发布的内容，包括文字，视频链接等，望大家理解！
+本视频预计发布时间为北京时间8月19日晚6点
+
+首先我们准备适量的小青椒和小米辣去蒂备用
+First, prepare moderate amount of "Xiaoqing Jiao"(Small Green Chilli Peppers) and "Xiaomi La"(Small Red Chilli Peppers), remove the pedicles and set aside
```
Comment (immoonancient) [inline subtitles/wang-gang/190818-qingjiao]:
```suggestion
First, prepare a moderate amount of "xiaoqingjiao" and "xiaomila" chili peppers, remove the pedicles and set aside
```
食材名称不用解释了

File: subtitles/wang-gang/190818-qingjiao [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+
+此辣椒碧绿短小辣度适中
+This pepper appears to be jade green, short and medium spicy
+
+是由湖南本土优质辣椒种子而结果
+It's the fruit of high-quality selected native Hunan Chilli Pepper
```
Comment (immoonancient) [inline subtitles/wang-gang/190818-qingjiao]:
```suggestion
They're fruits of premium Hunan local pepper seeds
```

File: subtitles/wang-gang/190818-qingjiao [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+
+怕辣的同学可以选择辣度较低的大青椒
+You may replace it with less spicy  "Daqing Jiao"(Large Green Pepper)
```
Comment (immoonancient) [inline subtitles/wang-gang/190818-qingjiao]:
```suggestion
You may replace it with the less spicy  "Daqingjiao" chili pepper
```

File: subtitles/wang-gang/190818-qingjiao [hunk 4]
```diff
@@ -0,0 +1,196 @@
...
+
+然后将辣椒清洗干净捞出备用
+Wash the peppers throughly, drain the water, then set aside
+
+下一步开始准备辅料
+Next, Prepare aromatics and other ingredients
```
Comment (immoonancient) [inline subtitles/wang-gang/190818-qingjiao]:
```suggestion
Next, prepare the "Fu Liao" ("auxiliary ingredients")
```
按照上次讨论的结果，“辅料” 不翻译

File: subtitles/wang-gang/190818-qingjiao [hunk 5]
```diff
@@ -0,0 +1,196 @@
...
+
+准备五花肉1小块切成厚片之后剁碎备用
+Cut a small chunk of pork belly into thick slices, then shred the meat and set aside
```
Comment (immoonancient) [inline subtitles/wang-gang/190818-qingjiao]:
```suggestion
Cut a small chunk of pork belly into thick slices, then mince it and set aside
```

File: subtitles/wang-gang/190818-qingjiao [hunk 6]
```diff
@@ -0,0 +1,196 @@
...
+
+加半肥瘦的五花肉爆香之后成菜会更有层次感
+# I split the sentence into two parts cuz I kinda suck...
+Stir fry the pork belly, with 50% percent of fat content, till aromatic.
+This step aims to enrich the abundance of favors of the finished dish.
+
+准备大蒜几颗拍散之后剁碎备用
+Shatter then mince some garlics, set aside
```
Comment (immoonancient) [inline subtitles/wang-gang/190818-qingjiao]:
```suggestion
Crush and mince some garlics, and set aside
```

File: subtitles/wang-gang/190818-qingjiao [hunk 7]
```diff
@@ -0,0 +1,196 @@
...
+Next, Prepare aromatics and other ingredients
+
+准备五花肉1小块切成厚片之后剁碎备用
+Cut a small chunk of pork belly into thick slices, then shred the meat and set aside
+
+加半肥瘦的五花肉爆香之后成菜会更有层次感
+# I split the sentence into two parts cuz I kinda suck...
+Stir fry the pork belly, with 50% percent of fat content, till aromatic.
+This step aims to enrich the abundance of favors of the finished dish.
+
```
Comment (vincenthychow) [inline subtitles/wang-gang/190818-qingjiao]: 建議
"Stir fry the half-fat pork belly till you can smell, so as to enrich the sense of layers of the product." 之類?

File: subtitles/wang-gang/190818-qingjiao [hunk 8]
```diff
@@ -0,0 +1,196 @@
...
+Note that garlic is irreplaceable as it serves as the soul of "Hupi Qingjiao"(Tiger Skin Green Pepper)
+
+下一步开始制作
+Next, start cooking
+
+首先我们把锅烧热
+First, heat the wok
+
+锅烧热之后将辣椒下锅
+After the wok heating up, put in the peppers
```
Comment (vincenthychow) [inline subtitles/wang-gang/190818-qingjiao]: After heating up the wok
或
After the wok heats up

File: subtitles/wang-gang/190818-qingjiao [hunk 9]
```diff
@@ -0,0 +1,196 @@
...
+
+然后烧小火将辣椒挤压翻炒6分钟左右
+Then stir fry on low heat for 6 minutes, squezz the peppers while frying
```
Comment (vincenthychow) [inline subtitles/wang-gang/190818-qingjiao]: squeeze or press ??

File: subtitles/wang-gang/190818-qingjiao [hunk 10]
```diff
@@ -0,0 +1,196 @@
...
+Wash the peppers throughly, drain the water, then set aside
+
+下一步开始准备辅料
+Next, Prepare aromatics and other ingredients
+
+准备五花肉1小块切成厚片之后剁碎备用
+Cut a small chunk of pork belly into thick slices, then shred the meat and set aside
+
+加半肥瘦的五花肉爆香之后成菜会更有层次感
+# I split the sentence into two parts cuz I kinda suck...
```
Comment (immoonancient) [inline subtitles/wang-gang/190818-qingjiao]: https://github.com/immoonancient/YTSubtitles/blob/005752b28905c889bc6c8c363c45c4686e9477e1/subtitles/wang-gang/190818-qingjiao#L28-L30

The dish can be better layered by cooking with half-fat half-lean pork belly

File: subtitles/wang-gang/190818-qingjiao [hunk 11]
```diff
@@ -0,0 +1,196 @@
...
+Then stir fry on low heat for 6 minutes, squezz the peppers while frying
+
+这一步的目的是将辣椒的表皮通过高温炒糊炒香
+This step serves to gelatinize and aromatize the exterior of the peppers through high temp.
+
+同学们切记这一步不能放油
+Remember to NEVER add oil during the process
+
+之后经过慢火挤压翻炒的辣椒才够香辣
+Also note that only after the low-heat "Squezz-Fry" procedure shall the peppers turn fragrant and spicy
```
Comment (vincenthychow) [inline subtitles/wang-gang/190818-qingjiao]: squeeze or press?

File: subtitles/wang-gang/190818-qingjiao (hunk 8 above)
Comment (immoonancient) [inline subtitles/wang-gang/190818-qingjiao]:
```suggestion
After the wok heats up, put in the peppers
```

File: subtitles/wang-gang/190818-qingjiao (hunk 9 above)
Comment (immoonancient) [inline subtitles/wang-gang/190818-qingjiao]:
```suggestion
Then stir fry on low heat for 6 minutes, and squeeze the peppers while frying
```

File: subtitles/wang-gang/190818-qingjiao [hunk 12]
```diff
@@ -0,0 +1,196 @@
...
+Add the cooked peppers
+
+下一步开始调味
+Next, seasoning the dish
+
+锅中加入适量的酱油从锅边淋入翻炒几下
+Add moderate amount of soysauce from the edge of the wok, stir fry
+
+当炒出锅汽之后再加入适量的食用盐
+Add some salt after "Guo Qi"(dunno how to do this...) emerges
```
Comment (vincenthychow) [inline subtitles/wang-gang/190818-qingjiao]: until you can smell the soysauce / until the soysauce starts evaporating 之類吧....

File: subtitles/wang-gang/190818-qingjiao [hunk 13]
```diff
@@ -0,0 +1,196 @@
...
+
+同学们食用盐可以略微多一点点这样才够味
+You may add a liitle bit more salt to promote the favor
```
Comment (vincenthychow) [inline subtitles/wang-gang/190818-qingjiao]: 用Enhance會否好點?

File: subtitles/wang-gang/190818-qingjiao [hunk 14]
```diff
@@ -0,0 +1,196 @@
...
+First, heat the wok
+
+锅烧热之后将辣椒下锅
+After the wok heating up, put in the peppers
+
+然后烧小火将辣椒挤压翻炒6分钟左右
+Then stir fry on low heat for 6 minutes, squezz the peppers while frying
+
+这一步的目的是将辣椒的表皮通过高温炒糊炒香
+This step serves to gelatinize and aromatize the exterior of the peppers through high temp.
```
Comment (immoonancient) [inline subtitles/wang-gang/190818-qingjiao]:
```suggestion
This step serves to gelatinize and aromatize the skin of the peppers through high temp.
```

File: subtitles/wang-gang/190818-qingjiao (hunk 11 above)
Comment (immoonancient) [inline subtitles/wang-gang/190818-qingjiao]:
```suggestion
Also note that only after the low-heat "Squeeze-Fry" procedure shall the peppers turn fragrant and spicy
```

File: subtitles/wang-gang/190818-qingjiao [hunk 15]
```diff
@@ -0,0 +1,196 @@
...
+
+同学们切记这一步不能放油
+Remember to NEVER add oil during the process
+
+之后经过慢火挤压翻炒的辣椒才够香辣
+Also note that only after the low-heat "Squezz-Fry" procedure shall the peppers turn fragrant and spicy
+
+当辣椒炒至表面焦香后取出备用
+Take out the peppers once the exterior turns brown and fragrant, set aside
```
Comment (immoonancient) [inline subtitles/wang-gang/190818-qingjiao]:
```suggestion
Take out the peppers once the skin turns brown and fragrant, and set aside
```

File: subtitles/wang-gang/190818-qingjiao [hunk 16]
```diff
@@ -0,0 +1,196 @@
...
+Medium-low spicy...
+
+因为我们比较能吃辣所有这个菜我没放醋
+We are used to spiciness so I did not add vineger 
+
+如果你不能吃辣的话可以放点醋
+You can add some if this is too spicy for you
+
+加点醋进去稀释一下它这个辣度
+Add some vineger can dilute down the spiciness
```
Comment (vincenthychow) [inline subtitles/wang-gang/190818-qingjiao]: Adding some vinegar...

File: subtitles/wang-gang/190818-qingjiao [hunk 17]
```diff
@@ -0,0 +1,196 @@
...
+We've done making the chilli peppers
+
+但是它辣不辣我们还不知道 要吃了才知道
+We are not sure weather it's spicy, gotta try it first
+
+辣度中等偏下
+Medium-low spicy...
+
+因为我们比较能吃辣所有这个菜我没放醋
+We are used to spiciness so I did not add vineger
```
Comment (vincenthychow) [inline subtitles/wang-gang/190818-qingjiao]: vinegar

File: subtitles/wang-gang/190818-qingjiao [hunk 18]
```diff
@@ -0,0 +1,196 @@
...
+Remember to NEVER add oil during the process
+
+之后经过慢火挤压翻炒的辣椒才够香辣
+Also note that only after the low-heat "Squezz-Fry" procedure shall the peppers turn fragrant and spicy
+
+当辣椒炒至表面焦香后取出备用
+Take out the peppers once the exterior turns brown and fragrant, set aside
+
+下一步开始炒肉末
+Next step, move to cook the ground meat
```
Comment (immoonancient) [inline subtitles/wang-gang/190818-qingjiao]:
```suggestion
Next step, move to cooking the ground meat
```

File: subtitles/wang-gang/190818-qingjiao [hunk 19]
```diff
@@ -0,0 +1,196 @@
...
+
+锅烧热之后加入适量的菜籽油下锅
+Add moderate amount of rapeseed oil after the wok heats up
```
Comment (immoonancient) [inline subtitles/wang-gang/190818-qingjiao]:
```suggestion
Add a moderate amount of rapeseed oil after the wok heats up
```

File: subtitles/wang-gang/190818-qingjiao [hunk 20]
```diff
@@ -0,0 +1,196 @@
...
+
+油温6成热之后将剁好的肉末下锅翻炒至干香
+When the oil is 60% done, add minced meat and stir fry till dried&aromatic
```
Comment (immoonancient) [inline subtitles/wang-gang/190818-qingjiao]:
```suggestion
When the oil is 60% hot, add minced meat and stir fry till dried & aromatic
```

File: subtitles/wang-gang/190818-qingjiao [hunk 21]
```diff
@@ -0,0 +1,196 @@
...
+
+当肉末炒至干香后加入蒜蓉翻炒均匀
+Then add the minced garlics, stir fry evenly
```
Comment (immoonancient) [inline subtitles/wang-gang/190818-qingjiao]:
```suggestion
Then add the minced garlic, and stir fry evenly
```

File: subtitles/wang-gang/190818-qingjiao [hunk 22]
```diff
@@ -0,0 +1,196 @@
...
+
+然后再加入干香的辣椒下锅
+Add the cooked peppers
+
+下一步开始调味
+Next, seasoning the dish
```
Comment (immoonancient) [inline subtitles/wang-gang/190818-qingjiao]:
```suggestion
Next, season the dish
```

File: subtitles/wang-gang/190818-qingjiao [hunk 23]
```diff
@@ -0,0 +1,196 @@
...
+
+锅中加入适量的酱油从锅边淋入翻炒几下
+Add moderate amount of soysauce from the edge of the wok, stir fry
```
Comment (immoonancient) [inline subtitles/wang-gang/190818-qingjiao]:
```suggestion
Add a moderate amount of soy sauce from the edge of the wok, and stir fry
```

File: subtitles/wang-gang/190818-qingjiao (hunk 12 above)
Comment (immoonancient) [inline subtitles/wang-gang/190818-qingjiao]:
```suggestion
Add some salt after "Guo Qi" (aka "Wok Hei" or "Breath of the Wok") emerges
```
https://en.wikipedia.org/wiki/Wok#Wok_hei

File: subtitles/wang-gang/190818-qingjiao (hunk 13 above)
Comment (immoonancient) [inline subtitles/wang-gang/190818-qingjiao]:
```suggestion
You may add a little bit more salt to promote the flavor
```

File: subtitles/wang-gang/190818-qingjiao [hunk 24]
```diff
@@ -0,0 +1,196 @@
...
+
+当炒出锅汽之后再加入适量的食用盐
+Add some salt after "Guo Qi"(dunno how to do this...) emerges
+
+同学们食用盐可以略微多一点点这样才够味
+You may add a liitle bit more salt to promote the favor
+
+再加入小半勺白糖翻炒均匀即可出锅装盘
+Add half a spoon of sugar and stir fry evenly, the dish is ready to serve
```
Comment (immoonancient) [inline subtitles/wang-gang/190818-qingjiao]:
```suggestion
Add half a spoon of sugar and stir fry evenly. The dish is ready to serve
```

File: subtitles/wang-gang/190818-qingjiao [hunk 25]
```diff
@@ -0,0 +1,196 @@
...
+
+一道香辣下饭的家常虎皮青椒就制作完成
+We have finished cooking a delicious home dish "Hupi Qingjiao"(Tiger Skin Green Pepper) 
+
+下面开始技术总结
+第一，不同辣椒的种类辣度也不一样，大家可以选择适合自己的辣椒
+第二，辣椒下锅必须小火慢炒，炒至辣椒表皮脱层干香即可
+第三，此菜可以不加肉末，但蒜蓉是必不可少
+Technical Summary:
+1: Chilli peppers varies in degree of spiciness, you can choose diffirent kinds according to your liking
```
Comment (immoonancient) [inline subtitles/wang-gang/190818-qingjiao]:
```suggestion
1: Chili peppers vary in the degrees of spiciness. You may choose according to your preference
```

File: subtitles/wang-gang/190818-qingjiao [hunk 26]
```diff
@@ -0,0 +1,196 @@
...
+一道香辣下饭的家常虎皮青椒就制作完成
+We have finished cooking a delicious home dish "Hupi Qingjiao"(Tiger Skin Green Pepper) 
+
+下面开始技术总结
+第一，不同辣椒的种类辣度也不一样，大家可以选择适合自己的辣椒
+第二，辣椒下锅必须小火慢炒，炒至辣椒表皮脱层干香即可
+第三，此菜可以不加肉末，但蒜蓉是必不可少
+Technical Summary:
+1: Chilli peppers varies in degree of spiciness, you can choose diffirent kinds according to your liking 
+2: Chilli peppers must be cooked on low heat, fry untill the outter layer detached, dried and aromatic
```
Comment (immoonancient) [inline subtitles/wang-gang/190818-qingjiao]:
```suggestion
2: Chili peppers must be cooked on low heat, and fried until the outer layer is detached, dried and aromatic
```

File: subtitles/wang-gang/190818-qingjiao [hunk 27]
```diff
@@ -0,0 +1,196 @@
...
+3: While ground meat counts as optional, garlics are a must to the dish
+
+下面开始试吃
+Next we'll try the dish~~
+
+经过一番制作我们这个辣椒已经做好了
+We've done making the chilli peppers
```
Comment (immoonancient) [inline subtitles/wang-gang/190818-qingjiao]:
```suggestion
We've done making the chili peppers
```

File: subtitles/wang-gang/190818-qingjiao (hunk 16 above)
Comment (immoonancient) [inline subtitles/wang-gang/190818-qingjiao]:
```suggestion
Add some vinegar can dilute the spiciness
```

File: subtitles/wang-gang/190818-qingjiao (hunk 17 above)
Comment (immoonancient) [inline subtitles/wang-gang/190818-qingjiao]:
```suggestion
We are used to spiciness so I did not add vinegar 
```

File: subtitles/wang-gang/190818-qingjiao [hunk 28]
```diff
@@ -0,0 +1,196 @@
...
+While cooking the peppers
+
+最需要注意的是这个辣椒
+It is important that the pepper...
+
+下完锅后一定要干炒把它吵到软
+...must be fried without oil till softened
+
+炒到干香
+...dried&aromatic
```
Comment (immoonancient) [inline subtitles/wang-gang/190818-qingjiao]:
```suggestion
...dried & aromatic
```

File: subtitles/wang-gang/190818-qingjiao [hunk 29]
```diff
@@ -0,0 +1,196 @@
...
+
+然后再放肉进去炒
+Then you can add meat for stir fry
+
+我们一般在饭店里做这个辣椒
+Usually in restaurant 
+
+基本上是宽油伺候
+we cook tthis dish with "broad oil"
```
Comment (immoonancient) [inline subtitles/wang-gang/190818-qingjiao]:
```suggestion
we cook this dish with "broad oil"
```

File: subtitles/wang-gang/190818-qingjiao [hunk 30]
```diff
@@ -0,0 +1,196 @@
...
+这个肉末非常香
+The meat is very fragrant
+
+喜欢吃肉末可以多加些
+You can add more if you are into minced meat
+
+其实这个是简配版的
+This is actually a simplified version
+
+高端版可以加点豆shi
```
Comment (immoonancient) [inline subtitles/wang-gang/190818-qingjiao]:
```suggestion
高端版可以加点豆豉
```

File: subtitles/wang-gang/190818-qingjiao [hunk 31]
```diff
@@ -0,0 +1,196 @@
...
+For improved version you can add fermented bean
```
Comment (immoonancient) [inline subtitles/wang-gang/190818-qingjiao]:
```suggestion
For improved version you can add douchi (aka fermented black bean sauce)
```

File: subtitles/wang-gang/190818-qingjiao (hunk 29 above)
Comment (vincenthychow) [inline subtitles/wang-gang/190818-qingjiao]: tthis ---> this

File: subtitles/wang-gang/190818-qingjiao [hunk 32]
```diff
@@ -0,0 +1,196 @@
...
+
+放牛肉粒
+and beef dices
+
+又香又辣
+both aromatic and spicy
+
+这个辣椒很适合这种土灶来炒
+This kind of pepper is suitable to fry with villege stove
```
Comment (immoonancient) [inline subtitles/wang-gang/190818-qingjiao]:
```suggestion
It's good to cook this kind of pepper on a firewood stove
```

File: subtitles/wang-gang/190818-qingjiao [hunk 33]
```diff
@@ -0,0 +1,196 @@
...
+
+就是用柴火慢慢的把这个辣椒煸香
+Use firewood to slowly stir-fry the pepper till aromatic
+
+非常下饭大家可以回去试一下
+Good company with rice, everybody can try in your house
```
Comment (immoonancient) [inline subtitles/wang-gang/190818-qingjiao]:
```suggestion
It's good company with rice. Everybody can try this at home
```
不知道算不算是和 "don't try this at home" 的梗相对应

File: subtitles/wang-gang/190818-qingjiao [hunk 34]
```diff
@@ -0,0 +1,196 @@
...
+It is important that the pepper...
+
+下完锅后一定要干炒把它吵到软
+...must be fried without oil till softened
+
+炒到干香
+...dried&aromatic
+
+然后再放肉进去炒
+Then you can add meat for stir fry
```
Comment (vincenthychow) [inline subtitles/wang-gang/190818-qingjiao]: Then you can add the meat for stir frying/ to stir fry

File: subtitles/wang-gang/190818-qingjiao (hunk 3 above)
Comment (NoraZheng) [reply subtitles/wang-gang/190818-qingjiao]: 大青椒 -> green bell pepper

File: subtitles/wang-gang/190818-qingjiao [hunk 35]
```diff
@@ -0,0 +1,196 @@
+虎皮青椒 - https://youtu.be/dvl2yq25nlU
+这是刚哥最新的视频私享链接，方便大家提前添加字幕，做菜过程文字稿在下面
+文字稿可能会和视频有微小出入，请大家以视频为主。然后就是由于刚哥的合同问题，请大家务必不要泄露或分享刚哥未在YouTube发布的内容，包括文字，视频链接等，望大家理解！
+本视频预计发布时间为北京时间8月19日晚6点
+
+首先我们准备适量的小青椒和小米辣去蒂备用
+First, prepare a moderate amount of "xiaoqingjiao" and "xiaomila" chili peppers, remove the pedicles and set aside
```
Comment (NoraZheng) [inline subtitles/wang-gang/190818-qingjiao]: pedicles -> stems

File: subtitles/wang-gang/190818-qingjiao [hunk 36]
```diff
@@ -0,0 +1,196 @@
...
+
+下一步开始准备辅料
+Next, Prepare aromatics and other ingredients
+
+准备五花肉1小块切成厚片之后剁碎备用
+Cut a small chunk of pork belly into thick slices, then shred the meat and set aside
+
+加半肥瘦的五花肉爆香之后成菜会更有层次感
+# I split the sentence into two parts cuz I kinda suck...
+Stir fry the pork belly, with 50% percent of fat content, till aromatic.
```
Comment (NoraZheng) [inline subtitles/wang-gang/190818-qingjiao]: pork belly that's 50% fat

File: subtitles/wang-gang/190818-qingjiao [hunk 37]
```diff
@@ -0,0 +1,196 @@
...
+This step aims to enrich the abundance of favors of the finished dish.
```
Comment (NoraZheng) [inline subtitles/wang-gang/190818-qingjiao]: favors -> flavor

File: subtitles/wang-gang/190818-qingjiao (hunk 6 above)
Comment (NoraZheng) [reply subtitles/wang-gang/190818-qingjiao]: a few cloves of garlic

File: subtitles/wang-gang/190818-qingjiao (hunk 14 above)
Comment (NoraZheng) [reply subtitles/wang-gang/190818-qingjiao]: gelatinize是胶化 应用caramalize

File: subtitles/wang-gang/190818-qingjiao (hunk 12 above)
Comment (NoraZheng) [reply subtitles/wang-gang/190818-qingjiao]: 锅气是餐饮界熟悉的概念 翻译成wok hei即可

File: subtitles/wang-gang/190818-qingjiao (hunk 4 above)
Comment (GUThe) [reply subtitles/wang-gang/190818-qingjiao]: ok

File: subtitles/wang-gang/190818-qingjiao (hunk 6 above)
Comment (GUThe) [reply subtitles/wang-gang/190818-qingjiao]: 好的

File: subtitles/wang-gang/190818-qingjiao (hunk 8 above)
Comment (GUThe) [reply subtitles/wang-gang/190818-qingjiao]: 嗯，谢了

**Comment by GUThe** [comment]
Thanks!

**Pull request merged**